packages/core/src/editor.tsx

```diff
@@ -36,6 +53,54 @@ export function Editor<K extends string = string>(props: EditorProps<K>) {
   }
 }
 
+export const EditorProvider: React.FunctionComponent<{
+  omitDragDropContext?: boolean
+}> = props => {
+  const store = React.useMemo(() => {
+    return createStore({
+      instances: {}
+    }).store
+    // We want to create the store only once
+    // TODO: add effects that handle changes to plugins and defaultPlugin (by dispatching an action)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
+  const children = <Provider store={store}>{props.children}</Provider>
+  if (props.omitDragDropContext) return children
+  return (
+    <DragDropContextProvider backend={HTML5Backend}>
+      {children}
+    </DragDropContextProvider>
+  )
+}
+
+export function EditorInstance<K extends string = string>(
+  props: EditorProps<K> & InnerEditorProps
+) {
+  const { store } = React.useContext(EditorContext)
```

Comment:
Is this still correct?

Reply:
yes, I think it is `{ store: { getState() }, dispatch() }` when using the EditorContext directly (not scoped)...